source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -38,13 +38,32 @@ OpenGLContext: class extends GpuContext {
 	_pointsShader: OpenGLMap
 	_meshShader: OpenGLMapMesh
 	_renderer: GLRenderer
-	_recycleBin := _RecycleBin new()
+	_recycleBinMonochrome: RecycleBin<OpenGLMonochrome>
```

Comment:
`_bin: RecycleBin<OpenGLPacked>`

`this _bin search(|image| image instanceOf(OpenGLMonochrome) && image size == size)`

Something like that.